app/services/doc_auth/error_generator.rb

```diff
@@ -215,8 +224,12 @@ def scan_for_unknown_alerts(response_info)
       unknown_fail_count
     end
 
-    def self.wrapped_general_error
-      { general: [Errors::GENERAL_ERROR], hints: true }
+    def self.general_error(_liveness_enabled)
```

Comment:
do we need the _liveness_enabled argument?

Reply:
@amirbey , we used to have a different general error message when there is liveness, my assumption is that we will have something for it down the road.